src/data/real/basic.lean

```diff
@@ -603,6 +603,53 @@ by rw [mul_comm, sqrt_mul' _ hx, mul_comm]
 @[simp] theorem sqrt_div (hx : 0 ≤ x) (y : ℝ) : sqrt (x / y) = sqrt x / sqrt y :=
 by rw [division_def, sqrt_mul hx, sqrt_inv]; refl
 
+/--
+This is used in proving the lemma that `sqrt` is a continuous function.
+The lemma, called `real.continuous_sqrt`, is located at the file `analysis.specific_limits`
+-/
+lemma abs_sqrt_sub_sqrt_le_sqrt_abs (x y : ℝ) : abs (sqrt x - sqrt y) ≤ sqrt (abs (x - y)) :=
+nonneg_le_nonneg_of_squares_le (sqrt_nonneg _) $
+begin
+  rw abs_mul_abs_self,
+  wlog h : y ≤ x using [x y],
+  -- The main goal, i.e., prove the goal with `y ≤ x`
+  { have eq₁ : sqrt (abs (x - y)) * sqrt (abs (x - y)) = x - y,
+      rw mul_self_sqrt (abs_nonneg _), apply abs_of_nonneg, linarith,
+    rw eq₁,
+    have eq₂ : (sqrt x - sqrt y) * (sqrt x - sqrt y) =
+      sqrt x * sqrt x - (2 * sqrt x * sqrt y - sqrt y * sqrt y), ring,
+    rw eq₂,
+    cases le_total 0 x with hx hx,
+    -- if `0 ≤ x`
+    rw mul_self_sqrt hx,
+    apply sub_le_sub_left,
+    cases le_total 0 y with hy hy,
+    { rw mul_self_sqrt hy,
+      exact calc
```

Comment:
You can actually just write `calc` instead of `exact calc`. You can then also put the first inequality on the same line.